pkgs/applications/graphics/imv/default.nix

```
@@ -17,10 +17,10 @@
 , libGLU
 , wayland
 # "libnsgif" is disabled until https://todo.sr.ht/~exec64/imv/55 is solved
-, withBackends ? [ "freeimage" "libtiff" "libjpeg" "libpng" "librsvg" "libheif" ]
-, freeimage
+, withBackends ? [ "libtiff" "libjpeg" "libpng" "librsvg" "libheif" ]
```

According to the linked issue in the comment above and the imv changelog 
> Limit libnsgif to <1.0.0 to fix build

we should be able to reintroduce `libnsgif` :+1: 
```suggestion
, withBackends ? [ "libtiff" "libjpeg" "libpng" "librsvg" "libheif" "libnsgif" ]
```
Also don't forget to remove the comment :+1: